pkg/builders/s2i/builder.go

```diff
@@ -356,6 +356,9 @@ func s2iScriptURL(ctx context.Context, cli DockerClient, image string) (string,
 			if err != nil {
 				return "", fmt.Errorf("cannot parse image name: %w", err)
 			}
+			if _, ok := ref.(name.Tag); ok {
```

Comment:
Could we ignore this for default (Red Hat) builder images?
It's not good look if there is warning in default configuration.

Reply:
Yes, I can add this check today.

Reply:
I already added it.